### DISCUSSION TITLE
connection

### DISCUSSION THREAD

**Discussion by fazil501001** [apernet/hysteria#267] [General]
hello sir tell me how can i write to you in telegram do you have a group

**Comment by tobyxdd** [comment]
Is there anything you can't post here?

**Reply by fazil501001** [reply]
very uncomfortable here

**Reply by fazil501001** [reply]
I would like to have a conversation

**Reply by fazil501001** [reply]
you can write me here in telegram @vintm1001

**Reply by tobyxdd** [reply]
What's this about?

**Reply by fazil501001** [reply]
you can write to me in telegram, I want to talk about the service, about your script

**Reply by fazil501001** [reply]
I want to talk about help in your scenario, I can’t block the country I’m in, but I need VPN to connect in my country but not go to local sites, so I decided to contact you and ask for help

**Reply by fazil501001** [reply]
more precisely, I turn on vpn in my country, but with vpn turned on, he should not go to sites in my country, how to block it
